<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FxaaDialog</class>
 <widget class="QDialog" name="FxaaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FXAA effect settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="subpix">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fxaaQualitySubpix can effect sharpness.&lt;/p&gt;&lt;p&gt;1.00 - upper limit (softer)&lt;/p&gt;&lt;p&gt;0.75 - default amount of filtering&lt;/p&gt;&lt;p&gt;0.50 - lower limit (sharper, less sub-pixel aliasing removal)&lt;/p&gt;&lt;p&gt;0.25 - almost off&lt;/p&gt;&lt;p&gt;0.00 - completely off&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>QualitySubpix</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QDoubleSpinBox" name="subpixValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>0.750000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="edgeThreshold">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fxaaQualityEdgeThreshold is the minimum amount of local contrast required to apply algorithm.&lt;/p&gt;&lt;p&gt;0.333 - too little (faster)&lt;/p&gt;&lt;p&gt;0.250 - low quality&lt;/p&gt;&lt;p&gt;0.166 - default&lt;/p&gt;&lt;p&gt;0.125 - high quality &lt;/p&gt;&lt;p&gt;0.063 - overkill (slower)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>QualityEdgeThreshold</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QDoubleSpinBox" name="edgeThresholdValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.125000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="edgeThresholdMin">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;fxaaQualityEdgeThresholdMin trims the algorithm from processing darks.&lt;/p&gt;&lt;p&gt;0.0833 - upper limit (default, the start of visible unfiltered edges)&lt;/p&gt;&lt;p&gt;0.0625 - high quality (faster)&lt;/p&gt;&lt;p&gt;0.0312 - visible limit (slower)&lt;/p&gt;&lt;p&gt;Special notes: due to the current implementation you&lt;/p&gt;&lt;p&gt;Likely want to set this to zero.&lt;/p&gt;&lt;p&gt;As colors that are mostly not-green&lt;/p&gt;&lt;p&gt;will appear very dark in the green channel!&lt;/p&gt;&lt;p&gt;Tune by looking at mostly non-green content,&lt;/p&gt;&lt;p&gt;then start at zero and increase until aliasing is a problem.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="title">
      <string>QualityEdgeThresholdMin</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QDoubleSpinBox" name="edgeThresholdMinValue">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-10000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.000100000000000</double>
        </property>
        <property name="value">
         <double>0.031200000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FxaaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FxaaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
